Makefile

```diff
@@ -115,10 +115,12 @@ test-short:
 	@go test ./... -short -timeout 1m
 .PHONY: test-short
 
-## test-e2e: Run end to end tests via knuu.
+## test-e2e: Run end to end tests via knuu. This requires a kube/config file to configure the kubernetes. Without
+## this, the test will not work
```

Comment:
[nit] the newline in this comment causes the second line to appear like a command.

<img width="1457" alt="Screenshot 2023-11-06 at 1 04 36 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/06269d8a-fc2d-43da-9fd6-b9faf647965a">


```suggestion
## test-e2e: Run end to end tests via knuu. This command requires a kube/config file to configure kubernetes.
```